pkg/router/router_test.go

```diff
@@ -165,7 +172,7 @@ func TestAdmissionEdgeCases(t *testing.T) {
 	}
 }
 
-func TestConfigTemplateExecution(t *testing.T) {
+func TestConfigTemplate(t *testing.T) {
```

Comment:
Is there a reason to change the method name?

Reply:
Yes, I think it reflects better what the test is doing. Before it was only testing the parsing of the template, not it also checks for some patterns in the generated config file.